docs/glossary.md

```diff
@@ -10,21 +10,23 @@ A Table of terms of Terms and Tools used by the Data Platform
  ### API
 Application Programming Interface, which is a software intermediary that allows two applications to talk to each other. Each time you use an app like Facebook, send an instant message, or check the weather on your phone, you're using an API.
  ### Athena
-[Athena](https://docs.aws.amazon.com/athena/latest/ug/what-is.html) is an AWS cloud-based service which enables you to query data in the data lake using SQL (Presto SQL). You can find out in [the Playbook article about how to query data in Athena](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/querying-and-analysing-data/querying-data-using-sql).<br/><br/>
+[Athena](https://docs.aws.amazon.com/athena/latest/ug/what-is.html) is an AWS cloud-based service which enables you to query data in the data lake using SQL (Presto SQL). You can find out in [the Playbook article about how to query data in Athena](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/querying-and-analysing-data/querying-data-using-sql).
  ### AWS
-[AWS (Amazon Web Services)](https://aws.amazon.com/what-is-aws/) is a comprehensive cloud platform offering over 200 different services including data storage, computing power, and tools Glue, Athena and Redshift which we use in the data platform.. AWS is the council’s primary cloud supplier and the primary environment in which we are building the data platform. <br/><br/>
+[AWS (Amazon Web Services)](https://aws.amazon.com/what-is-aws/) is a comprehensive cloud platform offering over 200 different services including data storage, computing power, and tools Glue, Athena and Redshift which we use in the data platform.. AWS is the council’s primary cloud supplier and the primary environment in which we are building the data platform.
  ### Bucket
 Amazon S3 stores data as objects within resources called "buckets" which are equivalent to folders. Object storage keeps the blocks of data that make up a file together and adds all of its associated metadata to that file. We use buckets to separate the different zones in the data lake.
  ### Cluster
+A cluster is a collection of servers that are connected together so that they can as a group provide the same service or accomplish the same task. We often use Clusters to spread the work that needs to be accomplished so that it can be completed faster but we also use them to provide resilience were one member of a cluster can fail so long as one member remains functional.
```

Comment:
```suggestion
A cluster is a collection of servers that are connected together so that they can as a group provide the same service or accomplish the same task. We often use clusters to spread the work that needs to be accomplished so that it can be completed faster but we also use them to provide resilience where one member of a cluster can fail so long as one member remains functional.
```